### PR TITLE
LPS-183690 Integration Tests in: ObjectEntryLocalServiceTest

### PR DESCRIPTION
**Pull request by pedro-oliveira446**
[LPS-183690](https://issues.liferay.com/browse/LPS-183690)

### CODE REVIEW

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Comment by carolmariaabb** [comment]
Removing the `Blocked` label as https://github.com/brianchandotcom/liferay-portal/pull/137615 was merged

@pedro-oliveira446 @p-albuquerque

**Review by p-albuquerque — CHANGES_REQUESTED** [review]
At first, I guess we can drop some commits, since them already is in Master:
https://github.com/liferay-objects/liferay-portal/pull/2225/commits/f1d4781838ab1386dadc3bf1b055ffbc847aeafc
https://github.com/liferay-objects/liferay-portal/pull/2225/commits/6dd918d72a8ada78deaecb362d1af40cfb0d55f2
https://github.com/liferay-objects/liferay-portal/pull/2225/commits/b9ed1bf4f762386a4462f35d4b8baff751859eee

Actually, I believe when you rebase this branch to deal with the conflicts, it already will be done.

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Comment by mateussgn** [comment]
Hey, @pedro-oliveira446, I suggest you divide the first commit in two, one for the `ObjectFieldTestUtil` creation, and another for the `ObjectFieldTestUtil` usage.
This makes it easier to maintain or revert something, if needed, in the future.

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java [hunk 1]
```diff
@@ -612,6 +619,144 @@ public void testAddObjectEntry() throws Exception {
...
+					).build());
+
+				_assertCount(1);
+
+				Assert.assertEquals(
+					"test",
+					MapUtil.getString(
+						_objectEntryLocalService.getValues(
+							objectEntry.getObjectEntryId()),
+						"encrypted"));
```
Comment (p-albuquerque) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java:660]: After this assert, I would like you try to get the encrypted value inside Dynamic ObjectEntry table, peharps through a DSL query, and assert it with encrypted value of the plain text that you assign in this field when execute the `_addObjectEntry`:

```java
Assert.assertEquals(
	_encryptor.encrypt((key, "test"),
         dslQueryResult... // value "test" exactly how it is stored in database, without pass by the decrypt process
)
```

**Review by p-albuquerque — CHANGES_REQUESTED** [review]

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Review by p-albuquerque — APPROVED** [review]

**Review by carolmariaabb — CHANGES_REQUESTED** [review]
Great job on this, @pedro-oliveira446!

Please fix the conflicts and see my comments below.

Could you please open a new PR with the requested changes and assign it to me? I'll close this one.

Thanks!

**Comment by carolmariaabb** [comment]
ci:close

**Pull request closed without merging**